packages/mui-material/src/Popper/Popper.tsx

```diff
@@ -72,10 +77,9 @@ const Popper = React.forwardRef(function Popper(
     ...other
   } = props;
 
-  const RootComponent = slots?.root ?? components?.Root;
+  const RootComponent = component ?? slots?.root ?? components?.Root;
```

Comment:
Is there any difference between this and letting the `styled` util handle the `component` prop as done in the other components?

Reply:
I think the result is the same, as Popper has no styles. I believe we should let `styled` handle this as in other components, for consistency. I don't know why it wasn't implemented like this from the start.